cli/client.go

```diff
@@ -1698,6 +1699,40 @@ var clientRestartTransfer = &cli.Command{
 	},
 }
 
+var clientDealSize = &cli.Command{
+	Name:      "deal-size",
```

Comment:
I'd call this `stat` in case we want to add other things here like CommP

Reply:
👍